sarif/cmdline/main.py

```diff
@@ -371,6 +372,15 @@ def _html(args):
     html_op.generate_html(input_files, args.image, output, multiple_file_output)
     return _check(input_files, args.check)
 
+def _emacs(args):
+    input_files = loader.load_sarif_files(*args.files_or_dirs)
+    input_files.init_default_line_number_1()
+    _init_path_prefix_stripping(input_files, args, strip_by_default=True)
+    _init_blame_filtering(input_files, args)
+    (output, multiple_file_output) = _prepare_output(input_files, args.output, ".txt")
+    emacs_op.generate_compile(input_files, output)
```

Comment:
Missing parameter here, `multiple_file_output` - fixed in https://github.com/microsoft/sarif-tools/pull/21